torch/ao/quantization/fx/backend_config/native.py

```diff
@@ -351,6 +351,42 @@ def _get_binary_op_configs():
     ]
 }
 
+def _get_bn_configs():
+    """ Get configs related to batchnorm
+    """
+    bn_configs = []
+    bn_to_fused_bn = {
+        torch.nn.BatchNorm2d: nni.BNReLU2d,
+        torch.nn.BatchNorm3d: nni.BNReLU3d,
+    }
+    for bn in [torch.nn.BatchNorm2d, torch.nn.BatchNorm3d]:
```

Comment:
Can you just iterate through `bn_to_fused_bn.keys()`?

Reply:
will fix in a future diff, I'm making some further changes down the road